action.yml

```
@@ -26,6 +26,9 @@ inputs:
   args:
     description: 'The docker run args'
     required: false
+  build-args:
+    description: 'The docker build args'
+    required: false
```

Could we call them options to stay consistent with docker - https://docs.docker.com/engine/reference/commandline/build/?

```suggestion
  build-opts:
    description: 'The docker build opts'
    required: false
```

---

sgtm, that name is better 👍 

Thinking about it, would it make sense to have something like https://github.com/ipfs/gateway-conformance/pull/125/files#diff-aa38a2216a456b6851247d357e99f30026306296d43bb3c1f843a6614524ef77R73 ?

We could pass actual Docker "build-args" and process them as they do in https://github.com/docker/build-push-action, we'd be consistent with docker's build-push-action this way.

---

That's pretty cool! I like it. Of course only if it's not too much trouble and it's easy to copy.

---

@galargh updated! I also slightly refactored the action to compute the image tag once instead of copying and pasting, I hope to make the commands more readable this way.

---

Example run in https://github.com/ipfs/gateway-conformance/actions/runs/5833455055/job/15821109884
("Run the tests" step)